TestApps/GtkOnMacTest/Program.cs

```diff
@@ -34,6 +34,7 @@ class MainClass
 	{
 		public static void Main (string[] args)
 		{
+			ObjCRuntime.Dlfcn.dlopen("/Library/Frameworks/Xamarin.Mac.framework/Versions/Current/lib/libxammac.dylib", 0);
```

Comment:
Why is this needed?

Reply:
Otherwise it fails to find the libxammac.dylib

Reply:
I think that's because we only copy Xamarin.Mac.dll:
https://github.com/mono/xwt/blob/275741a3ad577329fb8979285c4f8d2ef0112123/TestApps/GtkOnMacTest/GtkOnMacTest.csproj#L35

Reply:
see my change here: https://github.com/mono/xwt/commit/415e9e57dbd9f58c8607e628d788a9fc90ab7ad9
Afair that's required because of the project type.

Reply:
I've removed those changes - anyways they are unrelated to the PR, so you can apply them locally if you want to test